department-of-veterans-affairs/va.gov-cms
# VACMS-7597: CMS announcement init and styling

## Description

Closes #7597 .

## Testing done
Visited the unauthenticated drupal site and saw the new CMS announcement.

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/150411299-634d3c1e-19a4-4b0a-99a1-ae23c5336f32.png)


## QA steps

As a cms edmin
1. I can go to `/block/146?destination=/admin/structure/block/block-content` and edit the CMS announcement
2. Then I can make sure VAgovclaro is set as the default theme (or enable it if it is not)
3. Then I can confirm the CMS Announcement block is placed in the content region as the first block and enabled
2. Then I can log out and view the CMS announcement on the unauth page
3. Then validate Acceptance Criteria from issue
- [x] Content is editable by a non-technical admin
- [x] Accommodates a link that opens in a new tab


### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
